### PR TITLE
Add "Dans une galaxie près de chez vous" and "District 31" categories

### PR DESCRIPTION
**Pull request by frdeso**

### CODE REVIEW

File: qngng/qngng.py [hunk 1]
```diff
@@ -272,7 +272,9 @@ class _Format(enum.Enum):
 
 def _format_name(fullname, fmt=_Format.DEFAULT, with_middle_name_initial=True):
 
-    if fullname.middle_name is None:
+    if fullname.surname is None:
+        raw_name = '{}'.format(fullname.name)
+    elif fullname.middle_name is None:
```
Comment (eepp) [inline qngng/qngng.py]: This is becoming hard to read.

Can we have parts instead?

1. Start with an empty list named `parts`.
2. If there's a name, append it to the list.
3. If there's a middle name/initial, append it to the list (using the very sophisticated algorithm which is already there).
4. If there's a surname, append it to the list.
5. Let `raw_name` be `' '.join(parts)`.

**Review by eepp — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Much apprécié.

Can you compress the JSON files encore plus? A pass through `jq -c` devrait aider.

Marci!

File: qngng/qngng.py (hunk 1 above)
Comment (frdeso) [reply qngng/qngng.py]: done.

**Comment by eepp** [comment]
👌

**Pull request merged**